src/mock-doc/window.ts

```diff
@@ -846,6 +846,7 @@ export function cloneDocument(srcDoc: Document) {
 /**
  * Constrain setTimeout() to 1ms, but still async. Also
  * only allow setInterval() to fire once, also constrained to 1ms.
+ * @param win TODO
```

Comment:
did you mean to circle back and update this one? idk if we would be able to change the type from `any` to `MockWindow` somehow

Reply:
lol yup! We actually can't update that type to `any` just yet - there's some work to be done (partial exploratory/partial impl) to reconcile differences between `Window` and `MockWindow` in STENCIL-345. I added a comment to call this out for this function